lib/Sema/CSSimplify.cpp

```diff
@@ -9513,14 +9517,40 @@ performMemberLookup(ConstraintKind constraintKind, DeclNameRef memberName,
   if (isSingleUnlabeledPackExpansionTuple(instanceTy)) {
     auto elementTy = instanceTy->castTo<TupleType>()->getElementType(0);
     if (elementTy->is<TypeVariableType>()) {
-      MemberLookupResult result;
       result.OverallResult = MemberLookupResult::Unsolved;
       return result;
     }
   }
 
+  // Delay solving member constraint for unapplied methods
+  // where the base type has a conditional Sendable conformance
+  if (Context.LangOpts.hasFeature(Feature::InferSendableFromCaptures)) {
+    if (isPartialApplication(memberLocator)) {
+      auto sendableProtocol =
+          DC->getParentModule()->getASTContext().getProtocol(
```

Comment:
Nit: `DC->getParentModule()->getASTContext()` could be replaced with `Context.`.